### PR TITLE
Fix 2 issues of Get-NetIPAddress and CreateIfupConfigFile

### PR DESCRIPTION
**Pull request by huijuanzhao**
Issue 1, for the Get-NetIPAddress in SETUP-Inject-IP-Constants.ps1:
Before fix, only can run it in local server, but if run it in remote server, will get wrong IP/interface as it still get the local IP/interface.
After fix, "Invoke-Command -ComputerName $HvServer" can run successful both in local and remote server as there is parameter $HvServer which can specify the right server.

The test results before and after fix the issue:
Before fix the script:
Errors in output:
09/27/2019 01:43:10 : [ERROR] EXCEPTION: You cannot call a method on a null-valued expression.
09/27/2019 01:43:10 : [ERROR] Source: Line 1575 in script .\Libraries\CommonFunctions.psm1.

[LISAv2 Test Results Summary]
Test Run On : 09/27/2019 01:42:11
VHD Under Test : D:\RHEL-8.1.0-xxxx-x86_64-GEN2-A.vhdx
Initial Kernel Version: xxxx.el8.x86_64
Final Kernel Version : xxxx.el8.x86_64
Total Test Cases : 1 (0 Passed, 1 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:4

ID TestArea TestCaseName TestResult TestDuration(in minutes)
1 NETWORK              NET-INTERNAL                                                                      FAIL                 2.92 

After fix the script:
[LISAv2 Test Results Summary]
Test Run On : 09/27/2019 01:57:08
VHD Under Test : D:\RHEL-8.1.0-xxxx-x86_64-GEN2-A.vhdx
Initial Kernel Version: xxxx.el8.x86_64
Final Kernel Version : xxxx.el8.x86_64
Total Test Cases : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:4

ID TestArea TestCaseName TestResult TestDuration(in minutes)
1 NETWORK              NET-INTERNAL                                                                      PASS                 2.48

### CODE REVIEW

**Comment by huijuanzhao** [comment]
Issue 2, for the CreateIfupConfigFile in utils.sh:
Fix the CreateIfupConfigFile to be available both for RHEL_6, RHEL_7 and RHEL_8. Delete "NM_CONTROLLED=no" as there is only NetworkManager to handle the network as default in RHEL_8.
The "NM_CONTROLLED=no" means do not handle network with NetworkManager. This fix will not affect other version of RHEL, such as RHEL_6, RHEL_7. 

Before fix the script:
Test failed in RHEL_8.
Errors in output:
09/27/2019 02:54:06 : [ERROR] Could not ping from 10.10.10.1 to 10.10.10.2

After fix the script:
RHEL_8 test results:
[LISAv2 Test Results Summary]
Test Run On : 09/27/2019 08:22:10
VHD Under Test : D:\RHEL-8.1.0-xxxx-x86_64-GEN2-A.vhdx
Initial Kernel Version: xxxx.el8.x86_64
Final Kernel Version : xxxx.el8.x86_64
Total Test Cases : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:6

ID TestArea TestCaseName TestResult TestDuration(in minutes)

1 NETWORK              NET-PRIVATE                                                                       PASS                 3.62

RHEL_7 test results:
[LISAv2 Test Results Summary]
Test Run On : 09/27/2019 08:29:16
VHD Under Test : D:\RHEL-7.8-xxxx-x86_64-GEN2-A.vhdx
Initial Kernel Version: xxxx.el7.x86_64
Final Kernel Version : xxxx.el7.x86_64
Total Test Cases : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:5

ID TestArea TestCaseName TestResult TestDuration(in minutes)

1 NETWORK              NET-PRIVATE                                                                       PASS                 3.83

File: Testscripts/Windows/SETUP-Inject-IP-Constants.ps1 [hunk 1]
```diff
@@ -69,7 +69,12 @@ function Main {
         $privateIP = "fd00::4:10"
     }
 
-    $interfaces = (Get-NetIPAddress -addressFamily $addressFamily)
+	if ($addressFamily -eq "IPv4") {
```
Comment (juhlee-microsoft) [inline Testscripts/Windows/SETUP-Inject-IP-Constants.ps1]: It is better to merge above if-statement with the same condition.

**Review by juhlee-microsoft — CHANGES_REQUESTED** [review]

**Comment by juhlee-microsoft** [comment]
@huijuanzhao Can you reply to the comment?

File: Testscripts/Windows/SETUP-Inject-IP-Constants.ps1 (hunk 1 above)
Comment (huijuanzhao) [reply Testscripts/Windows/SETUP-Inject-IP-Constants.ps1]: Thanks for your suggestion, sorry to reply late as China National Day holiday.
Already merged above if-statement, but the check #code_quality failed, will check it and update later.

Comment (huijuanzhao) [reply Testscripts/Windows/SETUP-Inject-IP-Constants.ps1]: @juhlee-microsoft, already updated, please review, thank you.

Comment (juhlee-microsoft) [reply Testscripts/Windows/SETUP-Inject-IP-Constants.ps1]: I meant the line 64 and 72 are the same condition. Can you add your lines inside of the first if-statement?

**Review by juhlee-microsoft — APPROVED** [review]

**Pull request merged**